gui/default/syncthing/core/syncthingController.js

```diff
@@ -1402,6 +1402,11 @@ angular.module('syncthing.core')
                 autoNormalize: true
             };
             $scope.editingExisting = false;
+            $scope.newIgnores = {
+                edited: false,
```

Comment:
The use case I had in mind that required this was to explicitly overwrite a preexisting .stignore by saving an empty ignores modal (versus not using it at all -> keep existing file). However I am probably over-thinking this again - on the other hand does it hurt to track it?